src/LeafletMap/LeafletMap.tsx

```diff
@@ -16,6 +17,18 @@ const LeafletMap: React.FC<LeafletMapProps> = (props: LeafletMapProps) => {
     return JSON.parse(data) as GeoJsonObject;
   };
 
+  const onEachFeature = (feature: Feature<Geometry, any>, layer: Layer) => {
+    const popupContent = `
+      <div>
+        <p>Location: ${feature.properties?.loc[1]}, ${feature.properties?.loc[0]}</p>
+        <p>Acceleration: ${Number(feature.properties.value).toFixed(2)} (g)</p>
```

Comment:
if user has selected statistic `cov` this should read "CoV: {value}" instead of "Acceleration {value} (g)".
Use same number of significant figures as acceleration